### ISSUE TITLE
Network Import does not work

### ISSUE THREAD

**Issue by AndrewYRevell** [jevansbio/net2blend#1]
Hello

First, thanks so much for making this. It seems really useful. For some reason however, I cannot import the edge or vertex data
I get the error message above. I have tried under both Windows and Linux blender 2.9

![Screenshot from 2020-10-24 02-17-32](https://user-images.githubusercontent.com/43831513/97069555-1d455b00-159f-11eb-8ab9-a116a6fd997c.png)

**Comment by AndrewYRevell** [comment]
Never mind, I sort of figured it out. You have to use absolute paths. In a Linux environment it would be the screenshot below.
![Screenshot from 2020-10-24 02-32-30](https://user-images.githubusercontent.com/43831513/97069866-2fc09400-15a1-11eb-8444-44e980077e89.png)

**Comment by AndrewYRevell** [comment]
Then I got a separate issue. Something to do with the  curves. Only the nodes would be imported
![Screenshot from 2020-10-24 02-28-20](https://user-images.githubusercontent.com/43831513/97069872-3c44ec80-15a1-11eb-9d6d-4665687a64b2.png)

So then I commented out line #260. I am familiar with python, however, I am not familiar with using python in Blender. I am guessing in Blender 2.9 the bezier curves do not have use_uv_as_generated attribute anymore, whatever use_uv_as_generated means
![Screenshot from 2020-10-24 02-33-32](https://user-images.githubusercontent.com/43831513/97069885-54b50700-15a1-11eb-9a89-4a94f14b9631.png)



After that, it seems it all works! Hopefully. I will test this with actual real network data from my research. Thanks!

Workign example screenshots using the example code (R's igraph random graphs):
![Screenshot from 2020-10-24 02-36-24](https://user-images.githubusercontent.com/43831513/97069940-d6a53000-15a1-11eb-914f-1c4f309e9626.png)

![Screenshot from 2020-10-24 02-36-29](https://user-images.githubusercontent.com/43831513/97069943-dad14d80-15a1-11eb-9420-0609b3e790bb.png)
